### DISCUSSION TITLE
Connecting to remote cameras?

### DISCUSSION THREAD

**Discussion by TechSmartSolutions** [mrlt8/docker-wyze-bridge#700] [Q&A]
I have two Neighbors that want cameras, and I run my own frigate NVR.  They were impressed and how well frigate works and wanted in on that action. Since we are all using at least one of our cameras for exterior video surveillance we are cool with all having access to each others exterior feeds.  So question that comes up, can I run the docker wyze bridge at my house and connect to cameras that aren't on my local network?

**Comment by mrlt8** [comment]
The bridge should be able to pull the streams from all the cameras including shared cams. 

Just keep in mind that it will use additional data if it has to stream over the internet.
